src/ios/CDVInAppBrowser.m

```diff
@@ -1116,3 +1116,12 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
 
 @end
 
+/**
+ * Extend NSURLRequest to allow self-signed SSL certificates.
+ */
+@implementation NSURLRequest(DataController)
++ (BOOL)allowsAnyHTTPSCertificateForHost:(NSString *)host
+{
+    return YES;
+}
```

Comment:
We should be warning the user and allowing them to accept or reject the certificate.